superset/db_engine_specs.py

```diff
@@ -184,6 +186,28 @@ def select_star(cls, my_db, table_name, schema=None, limit=100,
             sql = sqlparse.format(sql, reindent=True)
         return sql
 
+    @classmethod
+    def modify_url_for_impersonation(cls, url, impersonate_user, username):
+        """
+        Modify the SQL Alchemy URL object with the user to impersonate if applicable.
+        :param url: SQLAlchemy URL object
+        :param impersonate_user: Bool indicating if impersonation is enabled
+        :param username: Effective username
+        """
+        if impersonate_user and username is not None:
```

Comment:
NIT: I'd go without the `is not None` as None evals to False

Reply:
Fixed